brain/CMakeLists.txt

```diff
@@ -8,16 +8,22 @@ source_group(\\ FILES CMakeLists.txt)
 
 set(BRAIN_PUBLIC_HEADERS
   circuit.h
-  morphology.h
   types.h
+  cell/morphology.h
+  cell/section.h
+  cell/soma.h
```

Comment:
I don't know much about glia to answer the question. Are morphology and gliamorphology different in terns of behaviours ? Sections etc. Or there will be a enumaration in morphology to state the type ?